components/IntlProvider.tsx

```diff
@@ -0,0 +1,43 @@
+'use client'
```

Comment:
Let's create a new folder called `providers` either in the root folder or under `components/providers` so we can more easily differentiate between `ui` and `functionality`.